### PR TITLE
Updating to RTM builds of xunit and Moq

### PR DESCRIPTION
**Pull request by ajaybhargavb**
@pranavkm

### CODE REVIEW

File: samples/StaticFileSample/project.json [hunk 1]
```diff
@@ -26,7 +26,10 @@
     ]
   },
   "tools": {
-    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*"
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": {
+      "version": "1.0.0-*",
+      "imports": "portable-net45+wp80+win8+wpa81+dnxcore50"
```
Comment (muratg) [inline samples/StaticFileSample/project.json]: Why did you need to modify the sample?

Comment (pranavkm) [inline samples/StaticFileSample/project.json]: Agreed, this looks super questionable.

Comment (ajaybhargavb) [inline samples/StaticFileSample/project.json]: This was needed because of a problem in my NuGet cache. Will remove.
